crud/ProcessAction.php

```diff
@@ -44,10 +49,16 @@ public function run($id)
                 $success = $file->transfer();
             }
             if ($success && $file->processed_on === null) {
-                $success = $model->parser->process($file);
+                try {
+                    $success = $model->parserConfiguration->process($file);
+                } catch (Yii\Base\Exception $e) {
+                    $message = $e->getMessage();
```

Comment:
@pradziszewski123 `$message` is basically a serialized exception, so it should also contain type of that exception.